.github/workflows/test.yml

```diff
@@ -61,7 +61,7 @@ jobs:
       # 4 - tests
       - name: Check with integration
         if: matrix.os == 'ubuntu-latest'
-        run: ./gradlew check -PintegrationTests=include -PverboseTests=ON
+        run: ./gradlew check -PintegrationTests=include
       - name: Check without integration
         if: matrix.os != 'ubuntu-latest'
-        run: ./gradlew check -PverboseTests=ON
+        run: ./gradlew check
```

Comment:
See the difference:
- before: https://github.com/ACINQ/lightning-kmp/runs/8012467463
- after: https://github.com/ACINQ/lightning-kmp/runs/8017194027

Latter build only shows failed test, much either to see what went wrong. Not sure what we gain with verbose logs for successful tests?

Reply:
This was done in #358